src/data_availability.rs

```diff
@@ -473,17 +417,17 @@ impl DataAvailability {
         if self.blocks.last().is_some() {
             if self
                 .blocks
-                .get(block.parent_hash.clone())
+                .get(block.parent_hash())
                 .unwrap_or(None)
                 .is_none()
             {
-                debug!(
+                warn!(
```

Comment:
I don't think this should be warn. A new node cathing up will log a warn for each block, this is a "normal" case when catching up